core/web/schema/type/spec.graphql

```diff
@@ -91,7 +91,7 @@ type OCR2Spec {
     transmitterID: String
     pluginType: String!
     pluginConfig: Map!
-    feedID: String!
+    feedID: String
```

Comment:
Does this require an operator ui update too?

Reply:
I'm not sure ... I don't see any non-nullability over there but it looks like some tests could be failing (after the previous PR).